setup.py

```diff
@@ -47,7 +47,7 @@
         'Programming Language :: Python :: Implementation :: PyPy',
     ],
     install_requires=[
-        'h11 ~= 0.7.0',  # means: 0.7.x where x >= 0
+        'h11 ~= 0.8.1',  # means: 0.8.x where x >= 0
```

Comment:
The comment here "where x >= 0" isn't strictly correct.
Should this just be a more relaxed `~= 0.8.0`, or does it strictly need to be >= 0.8.1?

Reply:
0.8.0 introduced errors which caused some wsproto tests to fail. 0.8.1 fixed the relevant bugs in h11 so it's required.